front/app/components/VoteControl/index.tsx

```diff
@@ -53,9 +53,10 @@ import { IParticipationContextType } from 'typings';
 interface IVoteComponent {
   active: boolean;
   enabled: boolean | null;
-  compact: boolean;
 }
 
+type Size = '1' | '2' | '3' | '4';
```

Comment:
Added an extra size (4) because I needed a smaller size than the current size 1. So what I did: made size 1 the small size I needed and moved all the other one number up (so 1 => 2, etc.)